apps/files/src/components/FilesAppBar.vue

```diff
@@ -114,18 +114,30 @@ export default {
         'Authorization': 'Bearer ' + this.getToken
       }
     },
-    activeRoute () {
-      if (this.$route.params.item) {
-        return this.getRoutes()
-      }
-
-      return false
-    },
     canUpload () {
       if (this.currentFolder === null) {
         return false
       }
       return this.currentFolder.canUpload()
+    },
+    breadcrumbs () {
+      let breadcrumbs = [{
+        index: 0,
+        text: this.$gettext('Home'),
+        to: '/files/list'
+      }]
+
+      if (this.$route.params.item) {
+        this.$route.params.item.split('/').forEach((element, index) => {
+          index++ // First item is home
+          breadcrumbs.push({
+            index,
+            text: element,
+            to: `${breadcrumbs[index - 1].to}/${element}`
```

Comment:
As we are exploding a URI coming from the router, I think we can forgo re-encoding it as it was done before.

Reply:
it has to be encoded - otherwise routing will not work - please bring back - THX